internal/database/repo_commits_changelists.go

```diff
@@ -18,6 +18,10 @@ type RepoCommitsChangelistsStore interface {
 	BatchInsertCommitSHAsWithPerforceChangelistID(context.Context, api.RepoID, []types.PerforceChangelist) error
 	// GetLatestForRepo will return the latest commit that has been mapped in the database.
 	GetLatestForRepo(ctx context.Context, repoID api.RepoID) (*types.RepoCommit, error)
+
+	// GetRepoCommit will return the mathcing row from the table for the given repo ID and the
+	// given changelsit ID.
```

Comment:
I wanted to change `changelsit` to `changelsif`, but then I remembered that SCIP is better and made the right choice.

```suggestion
	// GetRepoCommitChangelist will return the mathcing row from the table for the given repo ID and the
	// given changelist ID.
```